components/switchCampuses/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import { Button } from 'react-native-elements';
+import styles from './styles';
+
+export default class SwitchCampuses extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      region: {
+        latitude: 0,
+        longitude: 0,
+      },
+    };
+  }
+
+
+  async setLoyola() {
+    const state = await this.setState({
+      region: {
+        latitude: 45.458025,
+        longitude: -73.640192,
+      }
+    });
+    this.props.callBack(this.state.region);
+  }
```

Comment:
Instead of async/ await, you can use a nice React callBack method that is called once the state is set:
`functionSetLoyola() {
    this.setState({
      region: {
        latitude: 45.458025,
        longitude: -73.640192,
      }
    }, () => { this.props.callBack(this.state.region); });
  }`

Reply:
FIXED.